examples/ecs/query_bundle.rs

```diff
@@ -0,0 +1,69 @@
+use bevy::{
+    // diagnostic::LogDiagnosticsPlugin,
+    ecs::schedule::RunOnce,
+    log::{LogPlugin, LogSettings},
+    prelude::*,
+};
+
+fn main() {
+    App::build()
+        .insert_resource(LogSettings {
+            level: bevy::log::Level::DEBUG,
+            ..Default::default()
+        })
+        .add_plugin(LogPlugin)
+        .add_plugins(MinimalPlugins)
+        .insert_resource(bevy::app::ScheduleRunnerSettings::run_once())
+        .add_startup_system(setup.system())
+        .add_stage("diagnostic", SystemStage::single_threaded())
```

Comment:
i don't see the need for a new stage here. Can you just use the default "update" stage instead for simplicity?

Reply:
I wanted the printing to be sequential. That's why I did this. Although maybe I can make the app run single-threaded?

Reply:
And the `CoreStage::Update` is parallel by default, and I don't think I can change it from my end: 
```rust

            .add_stage(CoreStage::Update, SystemStage::parallel())
```

Reply:
Maybe this isn't a problem when one uses log instead of `dbg!` and `println!`? In any case, I'll make it resolved. Thanks for the feedback!